### DISCUSSION TITLE
.NET 6 and VS 2022 for macOS virtual environments

### DISCUSSION THREAD

**Discussion by AlexSrizhevskiy** [actions/virtual-environments#5194] [Installed software]
Hello. Could you provide an approximate release date for macOS virtual environments with .NET 6 and VS 2022?

**Comment by miketimofeev** [comment]
Hi @AlexSrizhevskiy!
We add VS2022 to macOS once it is released and .Net 6 will be bundled with the new VS version.

**Reply by qwertylolman** [reply]
so... vs2022 for mac was already released. where banana? :)

**Reply by miketimofeev** [reply]
VS2022 is already available on macOS-12 and will be available on macOS-11 as well starting next week
